ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/converters/RequestConverter.java

```diff
@@ -154,4 +157,20 @@ private static boolean isResourceIdRequired(final Ds3Request ds3Request) {
 
         return !RequestConverterUtil.isResourceSingleton(ds3Request.getResource());
     }
+
+    public static String getResponseType(final ImmutableList<Ds3ResponseCode> responseCodes) {
+        for (final Ds3ResponseCode responseCode : responseCodes) {
+            final int rc = responseCode.getCode();
+            if (rc >= 200 && rc < 300) {
+                for (final Ds3ResponseType responseType : responseCode.getDs3ResponseTypes()) {
+                    if (ConverterUtil.hasContent(responseType.getType()) && !responseType.getType().contentEquals("null")) {
+                        return StructHelper.getResponseTypeName(responseType.getType());
+                    } else if (ConverterUtil.hasContent(responseType.getComponentType()) && !responseType.getComponentType().contentEquals("null")) {
```

Comment:
its an "else if" - I want it to fall through and return an empty string if Success doesn't return anything to be parsed